packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -173,6 +181,14 @@ export function rebaseBranch<TChange>(
 			findCommonAncestor(targetCommit, targetHead) !== undefined,
 			0x676 /* target commit is not in target branch */,
 		);
+
+		logger?.sendTelemetryEvent({
```

Comment:
Have you considered using `logger?.sendPerformanceEvent(..)` instead? It seems like a good fit given the type of telemetry.